dist/profile/files/mirrorbrain/sync.sh

```diff
@@ -46,7 +46,7 @@ popd
 
 echo ">> Delivering bits to fallback"
 /srv/releases/populate-archives.sh
-/srv/releases/azure-sync.sh
+/srv/releases/batch-upload.bash
```

Comment:
```suggestion
/srv/releases/batch-upload.bash || true
```

The `batch-upload.sh` script in its current form returns a non-zero exit, not sure why, but this causes the script to stop executing immediately and fail to propagate changes to the mirrors

Reply:
i am not a big fan of || true, it makes it even more harder to detect when something get wrong, I prefer to use set t-e and exit 0 if nothing get wrong